### ISSUE TITLE
sending via UDP with callbacks causes console messages

### ISSUE THREAD

**Issue by chazomaticus** [phuesler/ain#32]
Running this code modified from the README:

```js
var SysLogger = require('ain2');
var logger = new SysLogger({tag: 'node-test-app', facility: 'daemon', hostname: '127.0.0.1', port: 514});

logger.warn('test', function () {});
```

results in this console message when you run it:

    [Function]

It seems like a syslog module shouldn't be calling console.log every time it's going to run a callback.

**Comment by phuesler** [comment]
Are you running this code inside a nodejs console?

**Comment by chazomaticus** [comment]
I'm running it via bash.

```shellsession
$ cat test.js
var SysLogger = require('ain2');
var logger = new SysLogger({tag: 'node-test-app', facility: 'daemon', hostname: '127.0.0.1', port: 514});

logger.warn('test', function () {});
$ node test.js
[Function]
$
```

**Comment by phuesler** [comment]
You are absolutely right, I left a [console.log](https://github.com/phuesler/ain/commit/8f23f8b9fb325753c0e59626045e432d5c1ec1f8) in there.

Fixed it and released a new [patch release v1.5.1](https://github.com/phuesler/ain/blob/v1.5.1/CHANGELOG.md#v151)

Thank you for catching it

**Comment by chazomaticus** [comment]
:+1:  Thanks!
